lib/createConfig.js

```diff
@@ -1,7 +1,28 @@
 const { merge } = require('webpack-merge');
+const path = require('path');
 const getBaseConfig = require('./getBaseConfig');
+const presets = require('./presets');
 
 module.exports = (commandName, configFragment = {}) => {
   const baseConfig = getBaseConfig(commandName);
+
+  if (commandName === 'webpack-prod') {
+    const cacheFolderPath = presets[commandName].getMfePath();
+    baseConfig.cache = {
+      type: 'filesystem',
+      cacheDirectory: path.resolve(cacheFolderPath, '.cache'),
```

Comment:
[clarification] Will MFEs need to add `.cache` to their `.gitignore` files so it doesn't get committed upstream to GitHub? If so, we should likely document this in the `@edx/frontend-build` README and in the GitHub release notes associated with this PR.

Likewise, we may want to write a brief ADR ([docs](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#adrs)) explaining the decision to add caching so the decision surrounding the caching change is captured for future reference. See existing ADRs [here](https://github.com/openedx/frontend-build/tree/master/docs).

Reply:
Good catch! Yes, we probably need to add `.cache`  folder to the git ignore file for MFEs